### PR TITLE
{bio}[foss/2020a] add plantcv 3.8.0, includes updated easyconfigs for dependencies

### PR DESCRIPTION
**Pull request by bjmoran**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Waking up Travis...

**Comment by bjmoran** [comment]
Thanks @boegel. The tests have finally worked!

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.17.2-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        'checksums': ['d7e56b9983e2e14716feaee560022baa8f92e97f36df7eee8b10e672383f694c'],
+    }),
+    ('networkx', '2.4', {
+        'checksums': ['f8f4ff0b6f96e4f9b16af6b84622597b5334bf9cae8cf9b2e42e7985d5c95c64'],
+    }),
+    ('PyWavelets', '1.1.1', {
+        'modulename': 'pywt',
+        'checksums': ['1a64b40f6acb4ffbaccce0545d7fc641744f95351f62e4c6aaa40549326008c9'],
+    }),
+    ('imageio', '2.6.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.17.2-foss-2020a-Python-3.8.2.eb:34]: @bjmoran The existing easyconfig for scikit-image 0.17.1 uses `imageio` 2.9.0, is there a reason you are downgrading that here?

Same question for `imread` (existing easyconfig has `imread` 0.7.4)

Comment (bjmoran) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.17.2-foss-2020a-Python-3.8.2.eb:34]: That scikit-image 0.17.1 easyconfig is newer than this pull request. I'll have a look at it.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node367.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/45138e21a35be4cc419e94ad7c8d4aa2 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
node119.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/4d561c80493e89aa885ed96ef2102a5d for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @bjmoran !

**Pull request merged**